tests/dataconverter/test_readers.py

```diff
@@ -51,32 +51,12 @@ def get_all_readers() -> List[ParameterSet]:
     """Scans through the reader list and returns them for pytest parametrization"""
     readers = []
 
-    # Explicitly removing ApmReader and EmNionReader because we need to add test data
     for reader in [get_reader(x) for x in get_names_of_all_readers()]:
-        if reader.__name__ in (
-            "ApmReader",
-            "EmOmReader",
-            "EmSpctrscpyReader",
-            "EmNionReader",
-        ):
-            readers.append(
-                pytest.param(
-                    reader, marks=pytest.mark.skip(reason="Missing test data.")
-                )
-            )
-        else:
-            readers.append(pytest.param(reader))
+        readers.append(pytest.param(reader))
```

Comment:
Will the reader test take place for all the readers like STM, XPS and so on?
We have ci/cd to test that readers, right? No?

Reply:
Here, we don't have the reader plugins installed, so the deleted part was not doing anything anymore.